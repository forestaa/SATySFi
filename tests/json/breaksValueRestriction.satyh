let-rec x = 0
