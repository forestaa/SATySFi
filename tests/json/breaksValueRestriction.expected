{
  "file": "breaksvaluerestriction.satyh",
  "range": {
    "start": { "line": 1, "character": 12 },
    "end": { "line": 1, "character": 13 }
  },
  "message":
    "! [Type Error] this expression breaks the value restriction;\n    it should be a syntactic function."
}
